XpringKit/PayID/PayIDClient.swift

```diff
@@ -29,8 +36,9 @@ public class PayIDClient {
   ///   - ach
   ///
   //  TODO: Link a canonical list at payid.org when available.
-  public init(network: String) {
+  public init(network: String, sessionManager: SessionManager = .default) {
```

Comment:
Is this a parameter now so we can fake network requests?

Reply:
Correct. This is a manager for `AlamoFire`, which is the networking library that OpenAPI uses.

TBH, I don't know a ton about how this actually works, so I'm mostly just following tutorials on how to mock swagger code.